docs/maya/glue.md

```diff
@@ -0,0 +1,145 @@
+# AdnGlue
+
+AdnGlue is a Maya node that can be used to attach multiple muscles together. This solver uses as input a series of simulated muscles and allows the user to define glue connections between those muscles by using a maximum distance value and painted maps.
+
+The maps required to control the AdnGlue solver can be painted using the standard Maya Paint tool.
+
+### How To Use
+
+The AdnGlue node can be applied to any set of geometries. For each geometry, their mesh at each frame is gathered and simulated to ensure the original distance between the neighboring geometries. This is achieved by creating attachments between each vertex of a geometry and the closest point on the surface of the other geometries. More aspects are then computed to ensure that the simulated output preserves the original shape, edge-lengths and volume of the simulated meshes. The output of the node is a mesh that combines all the input geometries together and adds the results of the solver.
+
+To create an AdnSkin deformer within a Maya scene, the following inputs must be provided:
+
+  - **Input Geometries** (IG): List of geometries to be simulated and attached together by the glue solver.
+
+> [!NOTE]
+> - Applying the AdnGlue node will automatically create a new mesh in your scene, which will display the results of the solver.
+
+
+The process to create an AdnGlue deformer is:
+
+1. Select the Input Geometries.
+2. Press ![Glue button](images/adn_glue.png){style="width:4%"} in the AdonisFX shelf or the Glue action in the AdonisFX’s Solvers menu, under the Create section. 
+3. A message in the terminal will notify you that AdnGlue has been created properly, meaning that it is ready to simulate with default settings. Check the next section to customize their configuration.
+
+## Attributes
+
+### Solver Attributes
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Enable**     | Boolean    | True    | ✓ | Flag to enable or disable the node computation. |
+| **Iterations** | Integer    | 3       | ✓ | Number of iterations that the solver will execute per simulation step. Greater values mean greater computational cost. Has a range of \[1, 10\]. The upper limit is soft, higher values can be used. |
+| **Stiffness**  | Float      | 5000.0  | ✓ | Defines the overall stiffness of the material to be used for the simulation. This value can be later overridden for each different aspect of the solver to fit creative needs. |
+| **Bypass**     | Boolean    | False   | ✓ | When set to True, this attribute makes the input geometry pass through the solver without any simulation being applied. This is very useful if you want to compare results between having and not having the glue simulation performed. |
```

Comment:
"When set to True, this attribute makes the input geometry pass through the solver without any simulation being applied". I was just wondering is we could reformulate this as we are not preserving the original separed geometries that go out of this node.

Maybe:

```suggestion
| **Bypass**     | Boolean    | False   | ✓ | When set to True, this attribute bypasses the simulation on the combined input geometries by sending the separated input meshes directly to the the generated output mesh without glue simulation. This is very useful if you want to compare results between having and not having the glue simulation performed. |
```

The above is just and example

Reply:
What about this;

"When set to True, this attribute makes the input geometry pass through the solver without any simulation being applied. The solver does only transfer the input geometries into the output combined mesh. This is very useful if you want to compare results between having and not having the glue simulation performed."

Reply:
I like it, yes!